library/cmac.c

```diff
@@ -779,6 +779,8 @@ static int cmac_test_subkeys( int verbose,
 
         if( verbose != 0 )
             mbedtls_printf( "passed\n" );
+
+        mbedtls_cipher_free( &ctx );
```

Comment:
Shouldn't this be outside the `for` loop? We'll free multiple times, and will use an uninitialised context after the first iteration.